### DISCUSSION TITLE
pixaroma 260+ Styles converted to .Json feel free to play

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/Fooocus#2685] [Ideas]
Hi, last week Pixaroma on youtube channel relased 260+ styles for  A1111 & Forge
https://www.youtube.com/watch?v=UyBnkojQdtU&t=72s
I have converted cvs styles to Json and manually edited to make compitable with Fooocus styles I added "pa" abbreviation to each style. This styles does not have preview yet, you have to build your own. I will highly recommend New art and craft, painting styles. Thery are amazing especcially if you mix with them some photography styles. I am not the developper of this styles. Huge thanks to Pixaroma.
I am not a developper i maybe make some mistakes.
if you are new to styles place the "sdxl_styles_pa.json" file to your \Fooocus\sdxl_styles
please see the attached file
[sdxl_styles_pa.json](https://github.com/lllyasviel/Fooocus/files/14822651/sdxl_styles_pa.json)

**Comment by poor7** [comment]
There are several more styles here.
https://github.com/Douleb/SDXL-A1111-Styles/blob/main/All%20in%20one%20Styles%20%28pro%20%2B%20experiemental%20GPT4%29%20850%2B/styles.csv

**Reply by wisdombox44** [reply]
Thank you i will definetly try them. That is a huge collection.

**Comment by bsdesigner** [comment]
> Hi, last week Pixaroma on youtube channel relased 260+ styles for A1111 & Forge https://www.youtube.com/watch?v=UyBnkojQdtU&t=72s I have converted cvs styles to Json and manually edited to make compitable with Fooocus styles I added "pa" abbreviation to each style. This styles does not have preview yet, you have to build your own. I will highly recommend New art and craft, painting styles. Thery are amazing especcially if you mix with them some photography styles. I am not the developper of this styles. Huge thanks to Pixaroma. I am not a developper i maybe make some mistakes. if you are new to styles place the "sdxl_styles_pa.json" file to your \Fooocus\sdxl_styles please see the attached file [sdxl_styles_pa.json](https://github.com/lllyasviel/Fooocus/files/14822651/sdxl_styles_pa.json)

Great job! Thank You!

**Comment by bsdesigner** [comment]
I tweaked the "sdxl_styles_pa.json" file a bit and made image icons for previewing. If anyone needs it, you are welcome to use it!
[Boy+Girl (Pixaroma 260+).zip](https://github.com/lllyasviel/Fooocus/files/14883333/Boy%2BGirl.Pixaroma.260%2B.zip)

**Comment by wisdombox44** [comment]
That is Great work many thanks for sharing with us. I am also still working on some styles. My experience with Pixaroma styles if we dived Pa styles on multiple files more easy to work inside foocus. Because single file with 260 styles moves top if you use them once. There is no button in fooocus to refresh syles in default order. if some one knows please reply. My recommendation is to divide them for multiple json files example pa-photography, pa-illustration, pa-fashion pa-craft_1, pa-craft_2. If we have styles manager in fooocus that would be so nice, some one shared few days back REQ Style Selector - JSON selector #2692 https://github.com/benjamin-bertram/StyleSelectorXL_JsonSelector for Forge and Automatic1111.  Thanks again

**Reply by bsdesigner** [reply]
I've already endorsed you: #2714

**Reply by bsdesigner** [reply]
I'm making seven styles right now, too:
- Architectural Styles;
- Design Styles;
- Artist Stiles;
- Cinematography Styles;
- Photography Styles;
- Sculptors Styles;
- Installation Styles.

I don't know how long it will take yet, but that there will be thousands of styles, that's for sure! So the question of how to manage them is becoming quite relevant....

**Reply by wisdombox44** [reply]
You May already know chat gpt is very efficient to create json code, always better to search manually and collect some key words for a specific style, and ask chat gpt to combine. İf you create a style manually sak chat gpt to check spellings errors and create a json file. So nice to  talk and share ideas with you. I hope to see your work soon. Thank You.
